src/movement/control_system.cpp

```diff
@@ -235,26 +235,51 @@ namespace robosub
                      * by adding in the new goal to the current goal to make it
                      * relative to the current state.
                      */
```

Comment:
This is still correct, all I've done is update the logic to work as expected for a relative change of zero.